ramani-maplibre/src/main/java/org/ramani/compose/Symbol.kt

```diff
@@ -28,10 +31,19 @@ fun Symbol(
     size: Float,
     color: String,
     isDraggable: Boolean,
+    onDrag: (LatLng) -> Unit = {},
```

Comment:
I am wondering here: instead of `onDrag()`, would it work to leverage hoisting with `center` above?

Reply:
Oh, so instead of a callback just updating a state object for the center? For my purposes, it's nice to have a callback because I do other things with the center coordinate of the symbol. One of the main use cases is drawing a route on the map by dragging a crosshairs symbol. Every time the coordinate for the symbol updates, we add a coordinate to the route.

Reply:
A related issue is that I'd like to be able to observe changes in the map position (center, current bounds, zoom), and I wasn't sure the best way to do that. I thought about adding callbacks similar to this, but it would make sense to treat both of these scenarios the same. What do you think? Also, do you have a Slack workspace for Ramani? If you are willing, that could be an easier way to bounce ideas back and forth.

Reply:
Hmm yeah, you're right that we already have a few other callbacks. I would like to think a bit about that before adding new callbacks everywhere 👍